k8-util/helm/fluvio-app/values.yaml

```diff
@@ -6,7 +6,7 @@ cloud: minikube
 loadBalancer:
   serviceAnnotations: {}
 service:
-  type: LoadBalancer
+  type: NodePort
```

Comment:
Should the helm chart be optimized for the local use-case or for the production use-case? In one of those cases the helm invocation will need to overwrite this.

Reply:
The helm chart is optimized for local environments